app/policies/admin_policy.rb

```diff
@@ -0,0 +1,40 @@
+class AdminPolicy
+  attr_reader :current_user, :model
+
+  def initialize(current_user, model)
+    raise Pundit::NotAuthorizedError, "must be logged in" unless current_user
+
+    @current_user = current_user
+    @user = model
+  end
+
+  def index?
+    @current_user.admin?
```

Comment:
actually better to initialise instance varaible, something like user_admin = @current_user.admin? and reuse it in all your methods